packages/providers/inpage-providers-hub/src/floatingButton/index.tsx

```
@@ -186,54 +205,59 @@
+              opacity: showCloseButton ? 1 : 0,
+              borderRadius: '9999px',
+              backgroundColor: '#fff',
+              border: '1px solid rgba(0, 0, 0, 0.1)',
+            }}
+            onClick={(event) => {
+              event.stopPropagation();
+              setIsShowCloseButton(false);
+              showCloseDialog();
+            }}
+          >
+            <svg width="16" height="16" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)